packages/select/src/components/query-list/queryList.tsx

```diff
@@ -199,6 +234,9 @@ export class QueryList<T> extends AbstractComponent<QueryListProps<T>, QueryList
     public constructor(props: QueryListProps<T>) {
         super(props);
 
+        // Generate unique ID for accessibility
+        this.listId = props.listId ?? `bp5-query-list-${Utils.uniqueId("ql")}`;
```

Comment:
Suggest removing the hardcoded version prefix and replacing with the following:
```suggestion
        this.listId = props.listId ?? Utils.uniqueId("bp-query-list");
```